src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = props => ( // eslint-disable-line
```

Comment:
If you are not using props for this component is perfectly fine to pass the functional component without a props argument, so that we don't have to rule out that line.